client/src/pages/dashboard/views/Staff.svelte

```diff
@@ -1 +1,16 @@
-<p>Manage Staff!</p>
+<script lang="ts">
+    import { Office } from '~model/office';
+    import { dashboardState } from '../stores/DashboardState';
+
+    let currentOffice: Office['id'] | null = null;
+
+    // eslint-disable-next-line no-unused-expressions, prefer-destructuring
+    $: $dashboardState.currentOffice ? currentOffice = $dashboardState.currentOffice : null;
+</script>
+
+{#if (currentOffice === null)}

```

Comment:
Parentheses are unnecessary here.
```suggestion
{#if currentOffice === null}
```